guides/user/get-started/index.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: single
+title:  "Get Started Using Spinnaker"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+> This article assumes you are new to Spinnaker. It contains basic information
+> about what to do after [installing Spinnaker](/setup/install/).
+
+## Before you get going...
+
+Whether you're an operator creating or managing a Spinnaker deployment, or an
+end user getting started with Spinnaker, here are some pointers to get you
+started.
+
+### Operators
+
+* If you want to install Spinnaker, [here you go](/setup/install/)
```

Comment:
Needs period. Also - consider "follow these instructions" vs. "here you go" ?

Reply:
I'm deliberately moving away from using periods at the ends of list items. Because I put the punk in punctuation. And because I know style is moving away from using periods where they're not really needed. They're needed as delimiters between sentences in a paragraph, but not necessarily at the ends of one-sentence (or phrase) list items.

But as I go looking for a link to provide you, from a particular style guide I use, I'm surprised to see that that guide still calls for periods. I'll will add them back in everywhere.